pkg/controller/eventing/inmemory/channel/reconcile_test.go

```diff
@@ -643,15 +643,15 @@ func makeVirtualService() *istiov1alpha3.VirtualService {
 		Spec: istiov1alpha3.VirtualServiceSpec{
 			Hosts: []string{
 				serviceAddress,
-				fmt.Sprintf("%s.%s.channels.cluster.local", cName, cNamespace),
+				fmt.Sprintf("%s.%s.channels", cName, cNamespace),
```

Comment:
I don't understand why we have this, since there's no DNS to back it up, but that's an issue for a different PR.